python/lsst/summit/utils/utils.py

```diff
@@ -481,3 +485,118 @@ def getSite():
 
     # we have failed
     raise ValueError('Location could not be determined')
+
+
+def getAltAzFromSkyPosition(skyPos, visitInfo):
+    """Get the alt/az from the position on the sky and the time and location
+    of the observation.
+
+    Parameters
+    ----------
+    skyPos : `lsst.geom.SpherePoint`
+        The position on the sky.
+    visitInfo : `lsst.afw.image.VisitInfo`
+        The visit info containing the time of the observation.
+
+    Returns
+    -------
+    alt : `lsst.geom.Angle`
+        The altitude.
+    az : `lsst.geom.Angle`
+        The azimuth.
+
+    Notes
+    -----
+    Currently, as only the AuxTel is on-sky, the location is assumed to be
+    ``AUXTEL_LOCATION``. When these utils are extended to the main telescope,
+    this will need correcting.
+    """
+    skyLocation = SkyCoord(skyPos.getRa().asRadians(), skyPos.getDec().asRadians(), unit=u.rad)
+    altAz = AltAz(obstime=visitInfo.date.toPython(), location=AUXTEL_LOCATION)
+    obsAltAz = skyLocation.transform_to(altAz)
+    alt = geom.Angle(obsAltAz.alt.degree, geom.degrees)
+    az = geom.Angle(obsAltAz.az.degree, geom.degrees)
+
+    return alt, az
+
+
+def getExpPositionOffset(exp1, exp2, useWcs=True):
+    """Get the change in sky position between two exposures.
+
+    Given two exposures, calculate the offset on the sky between the images.
+    If useWcs then use the (fitted or unfitted) skyOrigin from their WCSs, and
+    calculate the alt/az from the observation times, otherwise use the nominal
+    values in the exposures' visitInfos. Note that if using the visitInfo
+    values that for a given pointing the ra/dec will be ~identical, regardless
+    of whether astrometric fitting has been performed.
+
+    Values are given as exp1-exp2, are directional, and *not* given mod 360
+    so large values should be expected for small offsets depending on the
+    direction.
+
+    Parameters
+    ----------
+    exp1 : `lsst.afw.image.Exposure`
+        The first exposure.
+    exp2 : `lsst.afw.image.Exposure`
+        The second exposure.
+    useWcs : `bool`
+        Use the WCS for the ra/dec and alt/az if True, else use the nominal/
+        boresight values from the exposures' visitInfos.
+
+    Returns
+    -------
+    offsets : `lsst.pipe.base.Struct`
+        A struct contain the offsets:
+            - deltaRa
```

Comment:
Can you document the return types. Are they floats or Angle objects? I thought there was a documentation standard for Struct contents.

Reply:
Angles, and yes, I found an example and have added the docs.